resources/views/layouts/app.blade.php

```diff
@@ -9,12 +9,12 @@
     @yield ('meta')
 
     <link href="/css/app.css" rel="stylesheet">
-    @if (isset($gistlog) && $gistlog->isSecret())
+    @if (isset($gistlog) && $gistlog->isSecret() || !\App::environment('production'))
```

Comment:
dont' need the preceding slash on App in Blade

And, again, why are we dropping this on local?